docs/reference/monitoring/esms.asciidoc

```diff
@@ -0,0 +1,395 @@
+[role="xpack"]
+[[esms]]
+== {esms}
+
+The {esms} ({esms-init}) is a monitoring cluster on {ecloud}. Elastic provides
+and maintains {esms-init} for self-managed commercial customers. If you send
+your monitoring data to {esms-init}, it can also be used by Elastic support to
+provide better and faster incident resolution.
+
+NOTE: You must obtain your {esms-init} cluster URLs and credentials from the
+Elastic support team.
+
+[discrete]
+[[esms-elasticsearch]]
+=== Collecting monitoring data about {es}
+
+There are two methods for collecting and sending data about the health of your
```

Comment:
My main questions is whether or not we want to continue to document internal collection as an option at all. We definitely want people using Metricbeat by default at this point and I'm not sure that our customers would be well-served by presenting these two approaches as equivalent options. Thoughts?

Reply:
I agree that this should be revisited/removed when we clean up the internal collection methods (e.g. in https://github.com/elastic/elasticsearch/pull/47818).  I suggest that we move the content as-is and backport it appropriately, then perform the clean up on the appropriate subset of branches.

Reply:
Works for me. Thanks, @lcawl